student.rb

```diff
@@ -1,4 +1,5 @@
 require './person_class'
+require './classroom'
```

Comment:
- [OPTIONAL] I will highly recommend you to use `require_relative` here as we only use relative for external files as per Ruby convention.